pyhap/accessory.py

```diff
@@ -109,27 +109,46 @@ def pincode(self):
             self._pincode = util.generate_pincode()
         return self._pincode
 
-    def _set_services(self):
```

Comment:
I would really like backwards compatibility with _set_services. Do you think we can change the base implementation of it to not add an info service, so old clients can still use it? This way we will avoid a breaking change.

Reply:
I removed the breaking change and marked the `_set_service` method as `deprecated` instead. That way it should still work without issues.

Reply:
Have you tested? If not, I can do the testing for this and any necessary changes after this merge.

Reply:
Okay, then I am merging this and will try to test the backwards compatibility.